tests/countries/test_slovakia.py

```diff
@@ -9,44 +9,143 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-import unittest
-from datetime import date
-
-import holidays
-
-
-class TestSK(unittest.TestCase):
-    def setUp(self):
-        self.holidays = holidays.SK()
-
-    def test_2018(self):
-        # https://www.officeholidays.com/countries/slovakia/2018.php
-        self.assertIn(date(2018, 1, 1), self.holidays)
-        self.assertIn(date(2018, 1, 6), self.holidays)
-        self.assertIn(date(2018, 3, 30), self.holidays)
-        self.assertIn(date(2018, 5, 1), self.holidays)
-        self.assertIn(date(2018, 5, 8), self.holidays)
-        self.assertIn(date(2018, 4, 2), self.holidays)
-        self.assertIn(date(2018, 7, 5), self.holidays)
-        self.assertIn(date(2018, 8, 29), self.holidays)
-        self.assertIn(date(2018, 9, 1), self.holidays)
-        self.assertIn(date(2018, 9, 15), self.holidays)
-        self.assertIn(date(2018, 10, 30), self.holidays)
-        self.assertIn(date(2018, 11, 1), self.holidays)
-        self.assertIn(date(2018, 11, 17), self.holidays)
-        self.assertIn(date(2018, 12, 24), self.holidays)
-        self.assertIn(date(2018, 12, 25), self.holidays)
-        self.assertIn(date(2018, 12, 26), self.holidays)
-
-    def test_special_dates(self):
-        self.assertNotIn(date(1996, 5, 8), self.holidays)
-        self.assertIn(date(1997, 5, 8), self.holidays)
-
-        self.assertNotIn(date(2000, 11, 17), self.holidays)
-        self.assertIn(date(2001, 11, 17), self.holidays)
+
+from holidays.countries.slovakia import Slovakia, SK, SVK
+from tests.common import TestCase
+
+
+class TestSlovakia(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Slovakia)
+
+    def test_country_aliases(self):
+        self.assertCountryAliases(Slovakia, SK, SVK)
 
     def test_special_holidays(self):
-        self.assertIn(date(2018, 10, 30), self.holidays)
+        self.assertHoliday(
+            "2018-10-30",
+        )
+
+    def test_new_years_day(self):
+        self.assertHoliday(f"{year}-01-01" for year in range(1990, 2023))
+        self.assertHolidayNameInYears(
+            "Deň vzniku Slovenskej republiky", range(1990, 2023)
+        )
```

Comment:
```suggestion
        self.assertHolidaysName(
            "Deň vzniku Slovenskej republiky",
            (f"{year}-01-01" for year in range(1990, 2023)),
        )
```
Check it together by date and by name. And the same for rest of holidays.